### PR TITLE
👀 I don't think letting people do this is a good idea

### PR DESCRIPTION
**Pull request by Ikinon**
# Pull request

 - [x] I have read and agreed to the [CODE OF CONDUCT](https://github.com/CascadeBot/CascadeBot/blob/master/CODE_OF_CONDUCT.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [trello](https://trello.com/b/gUQehy4l/bot).
 - [x] I have tested all of my changes.
 - [ ] I have followed the style of the rest of the project.
 
## Added/Changed
Not allow people to try to use Moderation commands on the bot and sender

### CODE REVIEW

File: src/main/java/com/cascadebot/cascadebot/commands/moderation/KickCommand.java [hunk 1]
```diff
@@ -19,6 +19,14 @@ public void onCommand(Member sender, CommandContext context) {
             return;
         }
         Member targetMember = DiscordUtils.getMember(context.getMessage(0), context.getGuild());
+        if (targetMember.getUser() == sender.getUser()) {
+            context.replyWarning("Why would I kick you~");
```
Comment (binaryoverload) [inline src/main/java/com/cascadebot/cascadebot/commands/moderation/KickCommand.java]: Possibly a better phrasing?
```suggestion
            context.replyWarning("Why would you want to kick yourself~");
```

File: src/main/java/com/cascadebot/cascadebot/commands/moderation/KickCommand.java [hunk 2]
```diff
@@ -19,6 +19,14 @@ public void onCommand(Member sender, CommandContext context) {
...
+            return;
+        }
+        if (targetMember.getUser() == context.getSelfMember().getUser()) {
+            context.replyWarning("I'm not going to kick myself~");
```
Comment (binaryoverload) [inline src/main/java/com/cascadebot/cascadebot/commands/moderation/KickCommand.java]: ~~I'm sorry Dave, I'm afraid I can't let you do that~~ 

Maybe? Just a fun idea
```suggestion
            context.replyWarning("My programming forbids me to kick myself~");
```

File: src/main/java/com/cascadebot/cascadebot/commands/moderation/SoftBanCommand.java [hunk 3]
```diff
@@ -21,6 +21,14 @@ public void onCommand(Member sender, CommandContext context) {
             return;
         }
         Member targetMember = DiscordUtils.getMember(context.getMessage(0), context.getGuild());
+        if (targetMember.getUser() == sender.getUser()) {
+            context.replyWarning("Why would I soft ban you~");
```
Comment (binaryoverload) [inline src/main/java/com/cascadebot/cascadebot/commands/moderation/SoftBanCommand.java]: Same as the kick one

File: src/main/java/com/cascadebot/cascadebot/commands/moderation/SoftBanCommand.java [hunk 4]
```diff
@@ -21,6 +21,14 @@ public void onCommand(Member sender, CommandContext context) {
...
+            return;
+        }
+        if (targetMember.getUser() == context.getSelfMember()) {
+            context.replyWarning(("I'm not going to soft ban  myself~"));
```
Comment (binaryoverload) [inline src/main/java/com/cascadebot/cascadebot/commands/moderation/SoftBanCommand.java]: Double space

File: src/main/java/com/cascadebot/cascadebot/commands/moderation/SoftBanCommand.java [hunk 5]
```diff
@@ -21,6 +21,14 @@ public void onCommand(Member sender, CommandContext context) {
             return;
         }
         Member targetMember = DiscordUtils.getMember(context.getMessage(0), context.getGuild());
+        if (targetMember.getUser() == sender.getUser()) {
+            context.replyWarning("Why would you want to soft ban yourself~");
+            return;
+        }
+        if (targetMember.getUser() == context.getSelfMember()) {
+            context.replyWarning(("My programming forbids me to soft ban myself~"));
```
Comment (weeryan17) [inline src/main/java/com/cascadebot/cascadebot/commands/moderation/SoftBanCommand.java]: why the double `(` also why the `~` at the end of each line?

**Pull request merged**